docs/streamtypes/overview.md

```diff
@@ -1,9 +1,24 @@
 # StreamTypes
+Each [stream](../learn/glossary.md#streams) on Ceramic must be created using a specific [StreamType](../learn/glossary.md#streamtypes). Different StreamTypes have different APIs for performing [writes](../build/writes.md). To create and/or update streams, see the API page for the respective StreamTypes below.
```

Comment:
reads and writes! Caip10Links can be queried using the `fromAccount` method

Reply:
Is this true for all streamtypes or just CAIP10? I thought we wanted to keep the read APIs standardized so third-party developers don't need to care about what streamtype some other developer used when creating the data?

Reply:
`Ceramic.loadStream(<streamId>)` will work for any stream type.  But stream types may also expose additional special-purpose read APIs that are streamtype specific.  Deterministic document lookup within TileDocument can be thought of as another, and in fact we have plans to add a dedicated method for it: https://github.com/ceramicnetwork/js-ceramic/issues/1415

Reply:
Thanks for the additional context. That's good to know